.github/CONTRIBUTING.md

```diff
@@ -43,18 +43,36 @@ You can (and should) run our test suite using [*tox*].
 However, you’ll probably want a more traditional environment as well.
 We highly recommend to develop using the latest Python release because we try to take advantage of modern features whenever possible.
 
-Clone the *attrs* repository to your computer:
+First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
+
+Clone the fork to your computer:
 
 ```console
-$ git clone git@github.com:python-attrs/attrs.git
+$ git clone git@github.com:<your-username>/attrs.git
 ```
 
 Or if you prefer to use Git via HTTPS:
 
 ```console
-$ git clone https://github.com/python-attrs/attrs.git
+$ git clone https://github.com/<your-username>/attrs.git
+```
+
+Then add the *attrs* repository as *upstream* remote:
+
+```console
+$ git remote add upstream git@github.com:python-attrs/attrs.git
+```
+
+You can of course also rename the *origin* remote (your fork) to *fork* and add the repository as *origin*.
+
+The next step is to sync the upstream repository with your local copy:
+
+```console
+$ git fetch upstream
```

Comment:
```suggestion
$ git fetch upstream --prune-tags
```

Reply:
IIRC from reading the manpage two weeks ago, https://github.com/python-attrs/attrs/pull/1105/files#r1146115629 would make this redundant.

Also, `--prune-tags` kinda sets the expectation that you might be removing/force-pushing the upstream tags which is a bad tone. It's equivalent of force-pushing the `main` branch, we just don't do this for a good reason.

That said, you could apply this patch but it shouldn't be necessary in normal situation. Also, according to this guide, this is the first fetch of the upstream after adding it. So at this point, there are no tags in the repository fetched from upstream, hence there's nothing to prune from the local repo anyway.

Reply:
And in order to keep everything as simple as possible, options should be rationed ;-). So let's keep this as-is.